utils/test_memcheck.supp

```diff
@@ -640,3 +597,65 @@
    fun:_dl_init
    ...
 }
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Free
+   fun:free
+   ...
+   fun:dl_open_worker
+   fun:_dl_catch_error
+   fun:_dl_open
+   fun:dlopen_doit
+   fun:_dl_catch_error
+   fun:_dlerror_run
+   ...
+}
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Free
+   fun:free
+   ...
+   fun:ofi_get_core_info
+   fun:ofix_getinfo
+   fun:rxm_getinfo
+   ...
+}
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Free
+   fun:free
+   ...
+   fun:mca_base_component_repository_release
+   ...
+}
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Free
+   fun:free
+   ...
+   fun:__libc_freeres
+   fun:_vgnU_freeres
+   fun:__run_exit_handlers
+   ...
+}
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Leak
+   ...
+   fun:malloc
+   ...
+   fun:daos_debug_init
+   fun:dts_ctx_init
+   ...
+}
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Leak
+   match-leak-kinds: definite
+   fun:malloc
+   ...
+   fun:vos_db_init
```

Comment:
These last two seem troubling to me

Reply:
do you think these two suppressions can be one?

Reply:
I think if we really do have a bug in vos_db_init() then we need to fix it, although that's true to some extent for all the suppressions in this file.